docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+# docker-compose.yml
+services:
+
+  app:
+    image: "ghcr.io/remla23-team15/app:latest"
+    ports:
+      - 8080:80
```

Comment:
See comment below, `app` looks for `model-service` on port 8080, so this line should become:

- 8081:80